packages/requisitions/src/RequestRequisition/api/operations.graphql

```diff
@@ -10,120 +10,19 @@ fragment ItemWithStats on ItemNode {
   }
 }
 
-mutation insertRequest(
-  $storeId: String!
-  $input: InsertRequestRequisitionInput!
-) {
-  insertRequestRequisition(input: $input, storeId: $storeId) {
-    ... on RequisitionNode {
-      __typename
-      id
-      requisitionNumber
-    }
-    ... on InsertRequestRequisitionError {
-      __typename
-      error {
-        description
-        ... on OtherPartyNotASupplier {
-          __typename
-          description
-        }
-      }
-    }
-  }
-}
-
-mutation updateRequest(
-  $storeId: String!
-  $input: UpdateRequestRequisitionInput!
-) {
-  updateRequestRequisition(input: $input, storeId: $storeId) {
-    ... on RequisitionNode {
-      __typename
-      id
-      requisitionNumber
-    }
-    ... on UpdateRequestRequisitionError {
-      __typename
-      error {
-        description
-        ... on RecordNotFound {
-          __typename
-          description
-        }
-        ... on CannotEditRequisition {
-          __typename
-          description
-        }
-        ... on OtherPartyNotASupplier {
-          __typename
-          description
-        }
-      }
-    }
-  }
-}
-
-mutation deleteRequest(
-  $storeId: String!
-  $input: BatchRequestRequisitionInput!
-) {
-  batchRequestRequisition(storeId: $storeId, input: $input) {
-    deleteRequestRequisitions {
-      id
-      response {
-        ... on DeleteRequestRequisitionError {
-          __typename
-          error {
-            description
-            ... on RecordNotFound {
-              __typename
-              description
-            }
-            ... on CannotDeleteRequisitionWithLines {
-              __typename
-              description
-            }
-            ... on CannotEditRequisition {
-              __typename
-              description
-            }
-          }
-        }
-        ... on DeleteResponse {
-          id
-        }
-      }
-    }
-  }
-}
-
-fragment RequestLineChartData on ItemChartDataNode {
-  __typename
-  consumptionHistory {
-    nodes {
-      amc
-      consumption
-      isHistoric
-      date
-    }
-  }
-  suggestedQuantityCalculation {
-    stockOnHand
-    suggested
-    minimumStockOnHand
-    maximumStockOnHand
-    averageMonthlyConsumption
-  }
-  stockEvolution {
-    nodes {
-      stockOnHand
-      isHistoric
-      max
-      min
-      date
-    }
-  }
+fragment RequestRow on RequisitionNode {
```

Comment:
uh. yeah. Moved around things, so that fragments, queries and mutations were all co-located. 
It's easier to find things.. for me anyway.